### PR TITLE
PEP 3141: Fix typo in Number.Complex.imag() docstring

### PR DESCRIPTION
**Pull request by trcmohitmandokhot**
Made small typo changes per discussion in #2880. Scope of changes limited to reduce burden on historical PEP.

### CODE REVIEW

File: pep-3141.txt [hunk 1]
```diff
@@ -85,7 +85,7 @@ numbers are supported by this hierarchy. ::
 
         @abstractproperty
         def imag(self):
-            """Retrieve the real component of this number.
+            """Retrieve the complex component of this number.
```
Comment (Rosuav) [inline pep-3141.txt:88]: It's actually the *imaginary* component. A complex number has a real part and an imaginary part.

**Review by Rosuav — CHANGES_REQUESTED** [review]

File: pep-3141.txt (hunk 1 above)
Comment (trcmohitmandokhot) [reply pep-3141.txt:88]: Thanks for catching this. I will update the change.

**Review by Rosuav — APPROVED** [review]

**Comment by Rosuav** [comment]
Thanks for the fix, LGTM now.

**Review by CAM-Gerlach — APPROVED** [review]
LGTM, thanks @trcmohitmandokhot

**Comment by CAM-Gerlach** [comment]
@trcmohitmandokhot ?

**Comment by trcmohitmandokhot** [comment]
Jeez. I figured this was already closed and hence I deleted the repo. Let me fix this.

**Comment by trcmohitmandokhot** [comment]
@CAM-Gerlach - My apologies for the error. See #2882

**Comment by CAM-Gerlach** [comment]
Oops...

Just FYI, if it is merged, you'll see a big button that asks you if you want to delete your branch, and/or your fork, automatically.

**Comment by CAM-Gerlach** [comment]
It seems this one still works...

**Pull request merged**